java-shared-dependencies/unmanaged-dependency-check/pom.xml

```diff
@@ -0,0 +1,85 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>com.google.cloud</groupId>
+  <artifactId>unmanaged-dependency-check</artifactId>
+  <version>3.18.1-SNAPSHOT</version><!-- {x-version-update:google-cloud-shared-dependencies:current} -->
```

Comment:
Let's 0.0.1-SNAPSHOT and remove the release please annotation to ensure that we don't publish this artifact.

Reply:
done.